apps/backend/src/modules/users/user.service.ts

```diff
@@ -1,3 +1,4 @@
+import { bcrypt } from "~/libs/modules/bcrypt/bcrypt.js";
```

Comment:
The name `bcrypt` is conflicting with the library name, let's name the module `encrypt`